### PR TITLE
Allow setting Display Mode on AndroidManifest

### PR DESCRIPTION
**Pull request by andreban**
Haven't tested on proper devices yet. Do not merge.

### CODE REVIEW

File: androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java [hunk 1]
```diff
@@ -267,6 +272,26 @@ protected Uri getLaunchingUrl() {
         return TwaLauncher.CCT_FALLBACK_STRATEGY;
     }
 
+    protected TrustedWebActivityDisplayMode getDisplayMode() {
```
Comment (PEConn) [inline androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java:275]: I think this method should be in `LauncherActivityMetadata`. That class should do the parsing from Strings to domain types.

Comment (andreban) [reply androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java:275]: Cutout mode can have 3 different values, while sticky has 2. That leads to 6 different possible values.  We can either cover all values as strings or specify what be believe are the most common options and let developers override if needed.

Comment (andreban) [reply androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java:291]: changed.

**Comment by andreban** [comment]
Updated and tested on a proper device. @PEConn PTAL

**Pull request merged**